src/main/scala/io/iohk/ethereum/vmrunner/State.scala

```diff
@@ -0,0 +1,69 @@
+package io.iohk.ethereum.vmrunner
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.{Account, Address}
+import io.iohk.ethereum.vm._
+import WorldState.{PC, PR}
+
+object State {
+
+  val creatorAddress = Address(DataWord(0xabcdef)) //scalastyle:off magic.number
+
+  private var currentWorld = {
+    val creatorAcc = Account.Empty.updateBalance(BigInt(10).pow(9))
+    val creatorXAcc = XAccount(creatorAcc, "Creator", creatorAddress, Nil)
+    WorldState().saveXAccount(creatorAddress, creatorXAcc)
+  }
+
+  def world: WorldState = currentWorld
+
+
+  def createAccount(name: String, balance: BigInt, gas: BigInt, code: ByteString, abis: Seq[ABI]): (Address, PR) = {
+    val (newAddress, _) = world.newAddress(creatorAddress)
+    val tx = MockVmInput.transaction(creatorAddress, code, balance, gas)
+    val bh = MockVmInput.blockHeader
+
+    val context: PC = ProgramContext(tx, bh, world)
+    val intermediateResult: PR = VM.run(context)
+
+    val result: PR = if (intermediateResult.error.isDefined) intermediateResult else {
```

Comment:
I've also needed to add this logic. If you are not going to call `Ledger`(former TransactionExecutor) maybe it's a good idea to put this somewhere shared to be used from several places, what do you think?